### PR TITLE
Fix ParamCurveArclen::inv_arclen iteration count

### PR DESCRIPTION
**Pull request by chubei-oppen**

### CODE REVIEW

**Comment by cmyr** [comment]
Playing around, I agree that on larger curves the current implementation does strange things as the accuracy creeps towards 1.0, and we crash after falling through the loop at 1.0. I will defer to @raphlinus over whether this is our best solution. :)

File: src/param_curve.rs [hunk 1]
```diff
@@ -92,7 +92,7 @@ pub trait ParamCurveArclen: ParamCurve {
         let mut t_last = 0.0;
         let mut t0 = 0.0;
         let mut t1 = 1.0;
-        let n = (-accuracy.log2()).ceil();
+        let n = (self.arclen(accuracy) / accuracy).log2().ceil();
```
Comment (raphlinus) [inline src/param_curve.rs:95]: Yes, I think this is better logic. But to avoid a loop count of 0, I think it would be better to add a .max(1) on it.

Comment (chubei-oppen) [reply src/param_curve.rs]: Yes, this code panics on large accuracy values. Pull request updated.

**Review by cmyr — APPROVED** [review]
Thanks, I'll cut a release shortly.

**Pull request merged**